panda/src/audiotraits/openalAudioSound.cxx

```diff
@@ -745,7 +745,7 @@ set_3d_min_distance(PN_stdfloat dist) {
     _manager->make_current();
 
     alGetError(); // clear errors
-    alSourcef(_source,AL_REFERENCE_DISTANCE,_min_dist*_manager->audio_3d_get_distance_factor());
+    alSourcef(_source,AL_REFERENCE_DISTANCE,_min_dist);
```

Comment:
The way I understand, only the speed of sound needs to conform to whatever units are assumed. So if min/max distances (and source locations) are assumed to be e.g. in feet, and the speed of sound is scaled to feet, then the sound will travel distances in proper time.

The other way around would be not to multiply the speed of sound, but to divide every distance and source location by the distance factor.